x-pack/plugin/sql/jdbc/src/main/java/org/elasticsearch/xpack/sql/jdbc/JdbcHttpClient.java

```diff
@@ -68,14 +68,12 @@ Cursor query(String sql, List<SqlTypedParamValue> params, RequestMeta meta) thro
             fetch,
             TimeValue.timeValueMillis(meta.queryTimeoutInMs()),
             TimeValue.timeValueMillis(meta.pageTimeoutInMs()),
-            null,
```

Comment:
I've removed the filter and runtimeMapping because they are not used by JDBC and make serialization tricky for the filter (since it should be its own json-like structure).